src/annotations/AnnotationParser.php

```diff
@@ -27,6 +31,10 @@ class AnnotationParser
     const NAMESPACE_NAME = 6;
     const USE_CLAUSE = 11;
     const USE_CLAUSE_AS = 12;
+    const CLASS_USE_CLAUSE = 13;
```

Comment:
Please rename `CLASS_` in this in following 3 constants to `TRAIT_` because we're operating on trait-related tokens.